releasenotes/notes/enable_ovn-b84974f398242dcb.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Enabled ML2/OVN by default as checks preventing accidental migration
+    from ML2/OVS were added in kolla-ansible.
```

Comment:
If you are using a Neutron plugin other than ML2/OVN, set `kolla_enable_ovn` to `false`.

Reply:
Also point out that OVN distributed FIP is enabled, and how to disable it if necessary.

Reply:
Done.